Bottom: 9778dc4903a3d614f507a5c1867dedc54496681b
Top:    676b2a37e1a92dfa92e3d0c51cab14a68af0544b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 17:03:10 -0500

Remove container conf directory

Roles create it themselves


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index 396b842c..7ef3f40c 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -8,13 +8,6 @@
 - import_tasks: sshd_settings.yml
 - import_tasks: mariadb_settings.yml
 
-- name: base docker_container.conf dir
-  file:
-    state: directory
-    path: '{{ docker_restore_config_base_dir }}'
-    owner: root
-    group: root
-    mode: 'u=rwx,g=rx,o=rx'
 
 
 - name: Check if baculavirt selinux module exists
